runtime/onert/core/src/ir/train/TrainingInfo.cc

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "ir/train/TrainingInfo.h"
+
+namespace onert
+{
+namespace ir
+{
+namespace train
+{
+
+std::unique_ptr<TrainingInfo> TrainingInfo::createDefaultInfo()
+{
+  auto info = std::make_unique<TrainingInfo>();
+
+  info->setBatchSize(1);
+
+  onert::ir::train::LossInfo loss_info;
+  {
+    loss_info.loss_code = ir::train::LossCode::MeanSquaredError;
+    loss_info.reduction_type = ir::train::LossReductionType::SumOverBatchSize;
+  }
+  info->setLossInfo(loss_info);
+
+  onert::ir::train::OptimizerInfo opt_info;
+  {
+    opt_info.optim_code = ir::train::OptimizerCode::SGD;
+    opt_info.learning_rate = 0.001f;
+  }
+  info->setOptimizerInfo(opt_info);
```

Comment:
Should we initialize these variables? I think there is no need to redefine it here because there is already an initial value.

Reply:
I append my idea about the named constructor here - https://github.com/Samsung/ONE/pull/12340/files/21d8ef90f0277b3f9df5f822d26cfd8adf56372a#r1432279409. 

The initial constructor has initially "invalid" value by default. 
The named constructor returns a default "valid" info, to be used when training_info isn't given by a model or user.

Reply:
I removed the default valid training info after an offline discussion.